{
    "MITRE": {
        "CVE_data_meta": {
            "ASSIGNER": "secalert@redhat.com",
            "ID": "CVE-2021-3636",
            "STATE": "PUBLIC"
        },
        "affects": {
            "vendor": {
                "vendor_data": [
                    {
                        "product": {
                            "product_data": [
                                {
                                    "product_name": "openshift",
                                    "version": {
                                        "version_data": [
                                            {
                                                "version_value": "openshift 4.8"
                                            }
                                        ]
                                    }
                                }
                            ]
                        },
                        "vendor_name": "n/a"
                    }
                ]
            }
        },
        "data_format": "MITRE",
        "data_type": "CVE",
        "data_version": "4.0",
        "description": {
            "description_data": [
                {
                    "lang": "eng",
                    "value": "It was found in OpenShift, before version 4.8, that the generated certificate for the in-cluster Service CA, incorrectly included additional certificates. The Service CA is automatically mounted into all pods, allowing them to safely connect to trusted in-cluster services that present certificates signed by the trusted Service CA. The incorrect inclusion of additional CAs in this certificate would allow an attacker that compromises any of the additional CAs to masquerade as a trusted in-cluster service."
                }
            ]
        },
        "problemtype": {
            "problemtype_data": [
                {
                    "description": [
                        {
                            "lang": "eng",
                            "value": "CWE-295->CWE-287"
                        }
                    ]
                }
            ]
        },
        "references": {
            "reference_data": [
                {
                    "name": "https://bugzilla.redhat.com/show_bug.cgi?id=1978621",
                    "refsource": "MISC",
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1978621"
                }
            ]
        }
    }
}